notebooks/peerscout/peerscout-get-editor-pubmed-paper-ids.ipynb

```diff
@@ -127,22 +128,138 @@
     "print(\"existing_editor_ids_and_pubmed_url_df length: \", len(existing_editor_ids_and_pubmed_url_df))\n",
     "existing_editor_ids_and_pubmed_url_df.head(3)"
    ],
-   "outputs": [],
+   "outputs": [
```

Comment:
Oh, how come there are outputs. The pre-commit should have removed it.

See, that is why we perhaps do need the additional check that we had there before.

Reply:
And it's back again :D